parcels/particlefile/baseparticlefile.py

```diff
@@ -255,8 +320,22 @@ def add_metadata(self, name, message):
         else:
             setattr(self.dataset, name, message)
 
+    @abstractmethod
+    def get_pset_info_attributes(self):
+        """
+        :returns the main attributes of the pset_info.npy file.
+
+        Attention:
+        For ParticleSet structures other than SoA, and structures where ID != index, this has to be overridden.
+        """
+        return None
+
     def dump_dict_to_npy(self, data_dict, data_dict_once):
-        """Buffer data to set of temporary numpy files, using np.save"""
+        """
+        Buffer data to set of temporary numpy files, using np.save
+        :arg data_dict: a dict of data, each entry associating 'var_name' -> 2D numpy.ndarray of values (# particles x timesteps)
+        :arg data_dict_once:  a dict of data, each entry associating 'var_name' -> 2D numpy.ndarray of values (# particles x timesteps), only written once
```

Comment:
This is not true anymore. 
```suggestion
        :arg data_dict_once:  a dict of data, each entry associating 'var_name' -> 2D numpy.ndarray of values, only written once
```